src/main/java/com/redhat/devtools/lsp4ij/server/definition/LanguageServerDefinition.java

```diff
@@ -211,6 +250,37 @@ public <S extends LanguageServer> Launcher.Builder<S> createLauncherBuilder() {
                 });
     }
 
+    public <S extends LanguageServer> Launcher.Builder<S> createLauncherBuilder(LSPClientFeatures clientFeatures) {
```

Comment:
* Please remove the old `createLauncherBuilder()`
* use NotNull annotation: `createLauncherBuilder(NotNull LSPClientFeatures clientFeatures) `

Reply:
Done 🙇

Reply:
Thanks!